vscode-extension/src/extension.ts

```diff
@@ -1,29 +1,167 @@
+// tslint:disable:no-var-requires
+// tslint:disable:no-require-imports
+// tslint:disable:no-use-before-declare
+
 'use strict';
 // The module 'vscode' contains the VS Code extensibility API
 // Import the module and reference it with the alias vscode in your code below
 import * as vscode from 'vscode';
+const tooling = require('jsonforms-tooling');
 
 // this method is called when your extension is activated
 // your extension is activated the very first time the command is executed
-export function activate(context: vscode.ExtensionContext) {
+export const activate = (context: vscode.ExtensionContext) => {
 
-    // Use the console to output diagnostic information (console.log) and errors (console.error)
-    // This line of code will only be executed once when your extension is activated
-    console.log('Congratulations, your extension "vscode-extension" is now active!');
+  // The command has been defined in the package.json file
+  // Now provide the implementation of the command with  registerCommand
+  // The commandId parameter must match the command field in package.json
+  const createExampleProject = vscode.commands.registerCommand(
+    'extension.createExampleProject',
```

Comment:
Out of curiosity: are the 1st parameter and the `vscode` vs `theia` namespaces the only differences during command registration (we'll probably leave it as is in any case)?

Reply:
No it's not the only difference. Also the data that is provided to the `registerCommand` function is different. But apart from these two differences, the code is identical.